pkg/cli/start.go

```diff
@@ -607,9 +608,10 @@ If problems persist, please see %s.`
 			var buf bytes.Buffer
 			info := build.GetInfo()
 			tw := tabwriter.NewWriter(&buf, 2, 1, 2, ' ', 0)
-			fmt.Fprintf(tw, "CockroachDB node starting at %s (took %0.1fs)\n", timeutil.Now(), timeutil.Since(tBegin).Seconds())
-			fmt.Fprintf(tw, "build:\t%s %s @ %s (%s)\n", info.Distribution, info.Tag, info.Time, info.GoVersion)
-			fmt.Fprintf(tw, "webui:\t%s\n", serverCfg.AdminURL())
+			redact.Fprintf(tw, "CockroachDB node starting at %s (took %0.1fs)\n", timeutil.Now(), timeutil.Since(tBegin).Seconds())
```

Comment:
I'm assuming we're going to do something here to make this less of a footgun? (was discussed yesterday in the team meeting).

Reply:
Yes once https://github.com/cockroachdb/redact/pull/4 is merged I will rebase on top of that and remove the footgun here.

Reply:
Done.